scripts/ddp-RxTerms-script/src/index.ts

```diff
@@ -0,0 +1,53 @@
+import * as XLSX from "xlsx";
+import { RxNormDrugOption } from "./models";
+import { parseToHOCON, writeFile } from "./utils/common";
+
+const inputFileName = process.argv[2] || "RxTerms202202.txt";
+const outputFileName = process.argv[3] || "output.txt";
+
+const translations: string[] =  process.argv.filter(arg => ['en', 'es', 'ru'].includes(arg));
+
+const workBook = XLSX.readFile(inputFileName);
+
+const workSheet = workBook.Sheets[workBook.SheetNames[0]];
+
+const rxNormArray = XLSX.utils.sheet_to_json(workSheet) as RxNormDrugOption[];
+
+const filteredRxNormOptions = rxNormArray.map((drug) => ({
+  stableId: drug.RXCUI,
+  displayName: drug.DISPLAY_NAME
+}));
+
+const pickListOptions = filteredRxNormOptions.map((option) => ({
+  stableId: option.stableId,
+  optionLabelTemplate: {
+    templateType: "TEXT",
+    templateText: option.displayName
+  }
+}));
+
```

Comment:
I strongly suggest go directly from your rxNormArray to the list of picklistOptions that has the exact shape needed in the configuration and that is what you use to generate the JSON to export, e.g.:
```typescript
const pickListOptions = rxNormArray.map((drug) => ({
    stableId: drug.RXCUI.toString(),    // <--- I noticed we were exporting numbers instead of strings!
    optionLabelTemplate: {
        templateType: 'TEXT',
        templateText: '$drug_name',
        variables: [{
            name: 'drug_name',
            translations: languageCodes.map(lang => ({ language: lang, text: drug.DISPLAY_NAME.replace(/'/g, '') }))
        }]
    }
}));

Reply:
You can then do something like:

JSON.stringify({pickListOptions: pickListOptions}, null, 2) 

and that is the contents of the file. One iteration over data, no HOCON generation or parseHOCON necessary.

Reply:
Sounds good, thanks for that. I'm not familiar with ocon and wanted make sure it had the right format.